src/tablist.js

```diff
@@ -65,12 +71,18 @@ SideTabList.prototype = {
     document.addEventListener("dragover", e => this.onDragOver(e));
     document.addEventListener("drop", e => this.onDrop(e));
 
+    // Scroll to switch tabs
+    document.addEventListener("wheel", e => this.onScroll(e));
```

Comment:
Could we add that listener only when scrollTabs is activated? (This might require more code if you want to do it right, but you could go the lazy way and say this feature requires a restart).